sdf/1.7/particle_emitter.sdf

```diff
@@ -0,0 +1,109 @@
+<!-- Particle emitter -->
+<element name="particle_emitter" required="*">
+  <description>A particle emitter that can be used to describe fog, smoke, and dust.</description>
+
+  <attribute name="name" type="string" default="__default__" required="1">
+    <description>A unique name for the particle emitter.</description>
+  </attribute>
+
+  <attribute name="type" type="string" default="point" required="1">
+    <description>The type of a particle emitter. One of "box", "cylinder", "ellipsoid", or "point".</description>
+  </attribute>
+
+  <element name="emitting" type="bool" default="true" required="0">
+    <description>True indicates that the particle emitter should generate particles when loaded</description>
+  </element>
+
+  <element name="duration" type="double" default="0" required="0">
+    <description>The number of seconds the emitter is active. A value less than or equal to zero means infinite duration.</description>
+  </element>
+
+  <element name="size" type="vector3" default="1 1 1" required="0">
+    <description>
+    The size of the emitter where the particles are sampled.
+    Default value is (1, 1, 1).
+    Note that the interpretation of the emitter area varies
+    depending on the emmiter type:
+      - point: The area is ignored.
+      - box: The area is interpreted as width X height X depth.
+      - cylinder: The area is interpreted as the bounding box of the
+                  cilinder. The cylinder is oriented along the Z-axis.
+      - ellipsoid: The area is interpreted as the bounding box of an
+                   ellipsoid shaped area, i.e. a sphere or
+                   squashed-sphere area. The parameters are again
+                   identical to EM_BOX, except that the dimensions
+                   describe the widest points along each of the axes.
+    </description>
+  </element>
+
+  <element name="particle_size" type="vector3" default="1 1 1" required="0">
+    <description>The particle dimensions (width, height, depth).</description>
+  </element>
+
+  <element name="lifetime" type="double" default="5" required="0">
+    <description>The number of seconds each particle will ’live’ for before being destroyed. This value must be greater than zero.</description>
+  </element>
+
+  <element name="rate" type="double" default="10" required="0" min="0.0">
+    <description>The number of particles per second that should be emitted.</description>
+  </element>
+
+  <element name="min_velocity" type="double" default="1" required="0" min="0.0">
+    <description>Sets a minimum velocity for each particle (m/s).</description>
+  </element>
+
+  <element name="max_velocity" type="double" default="1" required="0" min="0.0">
+    <description>Sets a maximum velocity for each particle (m/s).</description>
+  </element>
+
+  <element name="scale_rate" type="double" default="1" required="0" min="0.0">
```

Comment:
suggestion to set default the scale rate to 0? so particles do not change in size by default

Reply:
Fixed in 944e13035dffed6df630545240e757c3d3706a36